### ISSUE TITLE
Size / Color / Shape of Map Icons / Quest objectives etc

### ISSUE THREAD

**Issue by Vexbane** [Questie/Questie#4659]
## Issues:

part of the problem I have is the sword just takes up too much space, even when you shrink it down, then all the other icons, are all brown-ish blending into the background of the map, so seeing them is always a issue.  classic codex was just small dots u could resize with colors like purple, yellow, pink etc so they stand out / easy to see.... you dont get the ' kill quest ' and 'loot' icons but it was worth it to me.  but in theory you could color code the dots with a (key legend) that explained what color was for what (posted in discord): https://discord.com/channels/263036731165638656/991572107773353994/1074496309278560286 

There is a downside to trying to do a "key legend" and certain colors having designations.  For example:  red dots = kill quests, green dots = $ / loot quest.  The issue of doing that would be that when you pick up lots of quests, there would be too many the same color.  The beauty of classic codex is that they were all colored different according to being on differ quests, so you could quickly see at a glance how many dots represented the quest you are on.  you would even shift left click those color dots to make them disappear if you wanted to focus on other dots / quests.  

In conclusion, If I had my choice, I would only use classic codex, just the way it was, regardless of losing the sword / money back icons etc.  It just looked a lot cleaner, I could see the dots better than questie icons.  But classic codex was not kept updated, so Im hoping that Questie can be the best of both and have maybe both options for those that like the swords / bag icons and those that like the dots.  

Thanks in advance for anything you are willing to do towards this aim.  

p.s. please put a #donations channel in the discord

**Comment by shagu** [comment]
I just got asked about the permissions via discord by @Muehe.
As the author of pfQuest, which is the origin of Classic Codex and also where the icons com from, you have my "Okay" to use them. I'm also totally fine with the included license change from MIT to GPL3 if that's required for Questie.

Cheers

**Comment by Vexbane** [comment]
@shagu Thank you so much!
